packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -222,7 +231,12 @@ const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>)
                     )}
                 </Helmet>
 
-                <Seo>
+                <Seo
+                    title={config.defaultSiteTitle}
+                    description="Your Description"
+                    noIndex={false}
+                    keywords="Your Keywords"
+                >
```

Comment:
Can you shine some more light on this change? Are these none optional params that we are doing our best to add the right values for?

Reply:
The issue here is that `with-layout.tsx` is TypeScript, while `seo/index.jsx` is in JavaScript. One option to fix the TypeScript errors is to provide default values. 

I agree that probably these are not the best default values, so I changed `seo/index.jsx` to `seo/index.tsx` and converted it to TypeScript.